pwiz_tools/Skyline/TestPerf/PerfImportResultsWatersIMSTest.cs

```diff
@@ -47,7 +47,7 @@ public class ImportWatersIMSTest : AbstractFunctionalTest
         public void WatersIMSImportTest()
         {
             Log.AddMemoryAppender();
-            TestFilesZip = GetPerfTestDataURL(@"PerfImportResultsWatersIMS.zip");
+            TestFilesZip = GetPerfTestDataURL(@"PerfImportResultsWatersIMSv2.zip"); // v2 has _func003.cdt file removed, to test our former assumption that lockmass would have IMS data if other functions did and vice verse

```

Comment:
Does this leave other tests still testing that the lockmass-with-IMS test still works?

Reply:
Hmm, good point. I could set it to use one data set or the other depending on day-of-year.

Reply:
You can delete the new dataset and instead (conditional on day of year) rename the func003.cdt file before the test and restore it afterward. Might have to handle restoring the file at the beginning if a previous run failed to restore it though.

Reply:
I think this is just easier. If that file somehow didn't get restored it would screw up any machine also running the release branch tests.